go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -220,6 +220,16 @@ func runOvnKube(ctx *cli.Context) error {
 
 	var watchFactory factory.Shutdownable
 	var masterWatchFactory *factory.WatchFactory
+	var libovsdbOvnNBClient, libovsdbOvnSBClient libovsdbclient.Client
+
+	if libovsdbOvnNBClient, err = libovsdb.NewNBClient(stopChan); err != nil {
```

Comment:
+1 I'm happy to leave the metrics stuff as ctl calls, I don't think it's great creating a ton of clients

Reply:
In general, we do *not* want the nodes to have long-running or frequent connections to either of the DBs.

Reply:
I've made sure there's no places we use a libovsdb client in a per node process

Reply:
But this is enabling the per node process to use them for metrics? I think this code cannot be merged as it will cause ovnkube-node to fail to rollout during upgrade.

I don't know why ovnkube-node tries to get ovn metrics. I spoke with @martinkennelly about this today. Something we need to go back and take a look at.